core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -510,9 +511,16 @@
    * LOAD_STATUS SUCCESS
    */
   public static final String STORE_LOADSTATUS_SUCCESS = "Success";
+
+  /**
+   * Default batch for data read in Columnar format
```

Comment:
1.Can you complement the unit info for "4096"
2.Can you explain why set "4096"?